nrf_rpc/nrf_rpc.c

```diff
@@ -726,6 +730,21 @@ static int init_packet_handle(struct header *hdr, const struct nrf_rpc_group **g
 	return 0;
 }
 
+static void abort_all_ops(void)
+{
+	NRF_RPC_WRN("Canceling all tasks.");
```

Comment:
Should it be a warn? It seems to be expected after user's action, how about using info level?